### DISCUSSION TITLE
Welcome to CppTemplateTutorial Discussions!

### DISCUSSION THREAD

**Discussion by wuye9036** [wuye9036/CppTemplateTutorial#65] [Announcements]
也不知道为啥discussion模板那么多废话反正已经开了

**Comment by vczh** [comment]
C++又多了好多新功能了，什么时候继续写？

**Comment by ichdream** [comment]
文章都放在一个md📃文件里了，感觉看起来非常不方便。
有个小建议：
- 在主README.md中放置目录，重点提示，篇首语等吗，目录添加到各章节的md链接🔗。
- 各章节再另外创建单独的md文件📃，单独存放。通过目录链接找到该章节。
- 图片的话，github貌似对国内的显示总是遇到障碍。图片加载不出来，不知道有没有好的办法解决？

**Comment by xinyuemoli** [comment]
求更新！！！！！！！！！！膜拜
![Suggestion](https://github.com/user-attachments/assets/25c3704a-e824-4653-adc7-7b9691dc3530)
